blog/_posts/2021-05-19-latest-stable.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "New 'latest-stable' branch in git repository"
+layout: archive
+author: Javier Lopez-Gomez
+---
+
+We have listened to you! We had users in the forum that were facing some issues building ROOT's `master` from source. However, in some circumstances the `master` branch might be temporarily broken on some platforms/compilers.
```

Comment:
The first paragraph should provide something akin to an "abstract", or a teaser, because it's what appears in the website front page.

Reply:
Fixed, thanks!